deploy_docker/merge_app.sh

```diff
@@ -39,16 +39,21 @@ if test -d "$INCOMING_DIR"; then
     printf "Add overlay-${i}/..."
     cp -r overlay-${i} "${STAGING_DIR}"
 
+    ln -s "${STAGING_DIR}/overlay-${i}" "${STAGING_DIR}/assets"
+
     # Handle image symlink creation if the overlay needs it
     if test -f "${STAGING_DIR}/overlay-${i}/images/create_links.sh"; then
     (
       cd "${STAGING_DIR}/overlay-${i}/images/" || exit 1
       printf "linking images..."
       source create_links.sh siteshir
     )
+    else
+       mkdir "${STAGING_DIR}/overlay-${i}/images"
     fi
 
-    linksAtEndOfHead="${linksAtEndOfHead}\n<link href=\"/${app_name}/overlay-${i}/image-overlays.css\" rel=\"stylesheet\">"
+    if test -d $INCOMING_DIR/images; then cp --no-clobber $INCOMING_DIR/images/*.* ${STAGING_DIR}/overlay-${i}/images; fi
+
```

Comment:
same with these, looks like it's including default images or something?

Reply:
This includes specialized `favicon.ico`,  `logo.png`, and potentially other files that may be part of the `simpleui.derivative`, but not part of the overlay-1.  If there's no specialized version, the generic simple-ui versions are used.